### ISSUE TITLE
App Build Automation

### ISSUE THREAD

**Issue by mcduffie** [unity-sds/unity-project-management#89]

**Comment by GodwinShen** [comment]
@mcduffie please create and/or link the work tickets for the acceptance criteria noted above.

**Comment by GodwinShen** [comment]
@mcduffie I noticed that the one work ticket (https://github.com/unity-sds/unity-app-build-trigger/issues/1) associated to this task is complete.  Is that the only ticket required to close this one out?  If not, can you please add any other related work tickets under the "Work Tickets" section here?

**Comment by GodwinShen** [comment]
Seems like we still need to complete https://github.com/unity-sds/unity-app-build-trigger/issues/2 before closing out this ticket, if so then I will add that ticket to the "Work Tickets" section here.

**Comment by mike-gangl** [comment]
@edwinsarkissian - we'll need to clean up the EC2 installs for the gitlab runners at somepoint. there is absolutely no version control happening and that's going to be an issue when trying to support dev/test/prod deployments - as what's installed on them will be a factor of _when_ they were installed.

**Comment by mike-gangl** [comment]
see https://github.com/unity-sds/unity-ads-deployment/blob/main/ci_cd/install_group_runner_x86_64_docker.tftpl#L171-L174 for examples fo what i'm talking about.

**Comment by edwinsarkissian** [comment]
Yes, I agree. I will need to deal with version control.  However, for now the problem is that registering image with the local image registry fails without any error message, and I have no idea what is causing it. I would have expected to see some error message or warning coming out of app-pack-gen or unity-app-gen. Unfortunately, I won't be able to spend more than 5% of my time on Unity, which I have already done.  I have to come back to this early next week.
